openmc/mgxs/library.py

```diff
@@ -717,3 +725,307 @@ def load_from_file(filename='mgxs', directory='mgxs'):
+                    xsdata.set_absorption(self.all_mgxs[id]['absorption'],
+                                          xs_type=xs_type,
+                                          subdomains=(id,))
+                if 'fission' in self.mgxs_types:
+                    xsdata.set_fission(self.all_mgxs[id]['fission'],
+                                       xs_type=xs_type, subdomains=(id,))
+                if 'kappa-fission' in self.mgxs_types:
+                    xsdata.set_k_fission(self.all_mgxs[id]['kappa-fission'],
+                                         xs_type=xs_type, subdomains=(id,))
+                if 'chi' in self.mgxs_types:
+                    xsdata.set_chi(self.all_mgxs[id]['chi'],
+                                   xs_type=xs_type, subdomains=(id,))
+                if 'nu-fission' in self.mgxs_types:
+                    xsdata.set_nu_fission(self.all_mgxs[id]['nu-fission'],
+                                          xs_type=xs_type,
+                                          subdomains=(id,))
+                # multiplicity requires scatter and nu-scatter
+                if ((('scatter matrix' in self.mgxs_types) and
+                     ('nu-scatter matrix' in self.mgxs_types))):
+                    xsdata.set_multiplicity(
+                        self.all_mgxs[id]['nu-scatter matrix'],
+                        self.all_mgxs[id]['scatter matrix'],
+                        xs_type=xs_type, subdomains=(id,))
+                    xsdata.multiplicity = np.nan_to_num(xsdata.multiplicity)
+                    using_multiplicity = True
+                else:
+                    using_multiplicity = False
+
+                if using_multiplicity:
+                    xsdata.set_scatter(self.all_mgxs[id]['nu-scatter matrix'],
+                                       xs_type=xs_type,
+                                       subdomains=(id,))
+                else:
+                    if 'nu-scatter matrix' in self.mgxs_types:
+                        xsdata.set_scatter(
+                            self.all_mgxs[id]['nu-scatter matrix'],
+                            xs_type=xs_type, subdomains=(id,))
+                        # Since we are not using multiplicity, then
+                        # scattering multiplication (nu-scatter) must be
+                        # accounted for approximately by using an adjusted
+                        # absorption cross section.
+                        # We can not do this with a transport x/s so check
+                        # for that.
+                        if 'total' in self.mgxs_types:
+                            xsdata.absorption = \
+                                np.subtract(xsdata.total,
+                                            np.sum(xsdata.scatter[0, :, :],
+                                                   axis=1))
+                        else:
+                            msg = "Absorption cross section must be " + \
```

Comment:
For consistency with the rest of the API, use single quotes around strings and remove the unnecessary `+` for concatenation.